neurons/validator/rewards/pipeline.py

```diff
@@ -0,0 +1,218 @@
+from typing import Dict, List, Optional
+
+import torch
+import bittensor as bt
+from loguru import logger
+
+from neurons.protocol import ModelType
+from neurons.validator.config import get_device
+from neurons.validator.rewards.models.blacklist import BlacklistFilter
+from neurons.validator.rewards.models.diversity import ModelDiversityRewardModel
+from neurons.validator.rewards.models.human import HumanValidationRewardModel
+from neurons.validator.rewards.models.image_reward import ImageRewardModel
+from neurons.validator.rewards.models.nsfw import NSFWRewardModel
+from neurons.validator.rewards.types import (
+    MaskedRewards,
+    PackedRewardModel,
+    RewardModelType,
+    AutomatedRewards,
+)
+
+ModelStorage = Dict[RewardModelType, PackedRewardModel]
+
+# Init Reward Models
+REWARD_MODELS: ModelStorage = {
+    RewardModelType.IMAGE: PackedRewardModel(
+        weight=0.8,
+        model=ImageRewardModel(),
+    ),
+    RewardModelType.DIVERSITY: PackedRewardModel(
+        weight=0.2,
```

Comment:
Comment out the diversity reward model. (Also, it should be called a Similarity model.)